### PR TITLE
Loadout Reworks

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
Reworks loadouts as pitched to Loremasters of factions.

Also added two new roles to NCR and re-added Venator, now renamed to Assassin, for Legion.

## Pre-Merge Checklist
- [ X ] You tested this on a local server.
- [ X ] This code did not runtime during testing.
- [ X ] You documented all of your changes.

## Changelog
:cl:
adds: Civilian Ranger role
adds: Off-Duty NCR role
tweak: Venator re-enabled and renamed to Assassin
tweak: Complete rework of all faction roles and loadouts as pitched to LMs via the google doc linked in coder-talk
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/ncr.dm [hunk 1]
```diff
@@ -1588,19 +1647,33 @@ Weapons		Service Rifle, Grease Gun, 9mm pistol, all good.
...
-	name = "Off-Duty"
-	belt = /obj/item/storage/belt/military/NCR_Bandolier
+
+
+/datum/job/ncr/f13ncroffduty
+	title = "NCR Off Duty"
+	flag = F13NCROFFDUTY
+	total_positions = 4
+	spawn_positions = 4
+	description = "You are the support element sent to assist the Camp Miller garrison. You are essential specialized support staff to help sustain the base via supply or specialized skills. You are not allowed to leave base unless given an explicit order by the CO or the current acting CO."
```
Comment (panzerr1944) [inline code/modules/jobs/job_types/ncr.dm:1660]: forgot to change the desc- this is the desc of the rear echelon.

**Comment by realgamerrrr** [comment]
Patrol Ranger/Assault Ranger is missing it's helmet, Veteran Ranger has had no changes to it's loadout, Patrol Ranger has a Sniper Rifle (which I assume is a mistake, could be a new weird Frontline Sniper build I guess).

File: code/__DEFINES/jobs.dm [hunk 2]
```diff
@@ -250,19 +251,19 @@
...
+#define JOB_DISPLAY_ORDER_HEAVYTROOPER 40
+#define JOB_DISPLAY_ORDER_DRILLSERGEANT 41
+#define JOB_DISPLAY_ORDER_SERGEANT 42
+#define JOB_DISPLAY_ORDER_CORPORAL 43
+#define JOB_DISPLAY_ORDER_COMBATMEDIC 44
+#define JOB_DISPLAY_ORDER_COMBATENGINEER 45
+#define JOB_DISPLAY_ORDER_TROOPER 46
+#define JOB_DISPLAY_ORDER_CONSCRIPT 47
+#define JOB_DISPLAY_ORDER_REAR_ECHELON 48
+#define JOB_DISPLAY_ORDER_VETRANGE 49
```
Comment (BadAtThisGame302) [inline code/__DEFINES/jobs.dm:264]: Can you fix this while you're at it as well? VETRANGE missing the R

File: code/modules/jobs/job_types/ncr.dm [hunk 3]
```diff
@@ -466,13 +483,6 @@ Weapons		Service Rifle, Grease Gun, 9mm pistol, all good.
 	outfit = /datum/outfit/job/ncr/f13representative
 	exp_requirements = 750
 
-	loadout_options = list(
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/ncr.dm:469]: Any reason why you remove all the fluff Rep loadouts for people who RP?

**Comment by YosemiteYam** [comment]
Does venator have the correct armor? Because I think there was another armor  listed under explorer/venator which had an odd description and stats as if it were slavemaster armor

**Comment by PigeonLord** [comment]
If I can make a suggestion - I think the Quill Scribes should have some basics in their Medical Belt at round start (ie A syringe, a Health Analyzer, a Bottle of Epinephrine, Sutures, Bandages, and Ointment)

File: code/modules/jobs/job_types/ncr.dm (hunk 3 above)
Comment (Stutternov) [reply code/modules/jobs/job_types/ncr.dm:469]: Thinks such as outfits should be in the loadout section as they don't provide protection. We don't need it to be in loadouts.

File: code/__DEFINES/jobs.dm (hunk 2 above)
Comment (Stutternov) [reply code/__DEFINES/jobs.dm:264]: Fixed

File: code/modules/jobs/job_types/ncr.dm (hunk 1 above)
Comment (Stutternov) [reply code/modules/jobs/job_types/ncr.dm:1660]: Fixed

**Comment by Stutternov** [comment]
> Does venator have the correct armor? Because I think there was another armor listed under explorer/venator which had an odd description and stats as if it were slavemaster armor, when it should actually be a modified (better) light explorer armor

It does, Legion assassin I was informed is to get heavily upgraded light armor instead of heavy gear. Thus its armor is a subset of Explorer armor but with much higher values for light armor.

**Comment by Stutternov** [comment]
> If I can make a suggestion - I think the Quill Scribes should have some basics in their Loadout Medical Belt (ie A syringe, a Health Analyzer, a Bottle of Epinephrine, Sutures, Bandages, and Ointment)

Added some of this. Mostly just extra basic healing gear and anti-rad medication.

**Comment by YosemiteYam** [comment]
I feel like replacing the venators upgraded sniper rifle with an amr in the "Deadeye" loadout doesn't exactly address the issue that legion has in how it has no actual long range roles while the NCR has 3 snipers on their rangers, since the amr isn't long range and the explorers have the acog range scopes. Maybe instead of an AMR you just let them keep their unique sniper rifle so legion has 1 actual sniper loadout? Makes more sense that the venators sharpshooter loadout is an actual sniper too and not an amr that has only close range scoping capabilities

**Comment by Stutternov** [comment]
> I feel like replacing the venators upgraded sniper rifle with an amr in the "Deadeye" loadout doesn't exactly address the issue that legion has in how it has no actual long range roles while the NCR has 3 snipers on their rangers, since the amr isn't long range and the explorers have the acog range scopes. Maybe instead of an AMR you just let them keep their unique sniper rifle so legion has 1 actual sniper loadout? Makes more sense that the venators sharpshooter loadout is an actual sniper too and not an amr that has only close range scoping capabilities

The Legion has multiple roles with scopes and rifles now, it’s simply an option. Similarly the issue was damage drop off on .44 was pistol light, this 2 tiles before damage started dropping. Causing half the issue of this. This has been amended separately. The reason for the AMR is the counter to the HT, thus why the role was enabled again.

**Pull request merged**